### ISSUE TITLE
Could not get registry endpoint from configuration: no registry with prefix

### ISSUE THREAD

**Issue by nishit93-hub** [argoproj-labs/argocd-image-updater#370]
Hi there,

I am using the Jfrog registry for docker images. This is my docker image `sn1-r620-a04-03.puretec.purestorage.com/springboot/maven`

I am below error in logs 

```
time="2022-01-29T21:21:59Z" level=info msg="Processing results: applications=1 images_considered=1 images_skipped=0 images_updated=0 errors=1"
time="2022-01-29T21:21:59Z" level=error msg="Could not get registry endpoint from configuration: no registry with prefix 'sn1-r620-a04-03.puretec.purestorage.com' configured" alias= application=maven-app image_name=springboot/maven image_tag=2.0 registry=sn1-r620-a04-03.puretec.purestorage.com
```

Even though I have added prefixes in registries.conf file

```
apiVersion: v1
 kind: ConfigMap
 metadata:
 labels:
 app.kubernetes.io/name: argocd-image-updater-config
 app.kubernetes.io/part-of: argocd-image-updater
 name: argocd-image-updater-config
data:
  registries.conf: |
    registries:  
    - name: jfrog Container Registry
      api_url: https://sn1-r620-a04-03.puretec.purestorage.com
      prefix: sn1-r620-a04-03.puretec.purestorage.com
      ping: no
      credentials: pullsecret:argocd/regcred
```
While testing the image from inside the argo image updater pod. I am getting this error

```
~/config $ argocd-image-updater test sn1-r620-a04-03.puretec.purestorage.com/springboot/maven --registries-conf registries.conf 
INFO[0000] getting image                                 image_name=springboot/maven registry=sn1-r620-a04-03.puretec.purestorage.com
DEBU[0000] rate limiting is disabled                     prefix=sn1-r620-a04-03.puretec.purestorage.com registry="https://sn1-r620-a04-03.puretec.purestorage.com"
INFO[0000] Loaded 1 registry configurations from registries.conf 
WARN[0000] cannot user K8s credentials without Kubernetes client  registry="https://sn1-r620-a04-03.puretec.purestorage.com"
FATA[0000] could not set registry credentials: could not fetch image tags .
```
Please guide what i am missing or what wrong i have mentioned.

**Comment by janpieper** [comment]
Can you please check what version of the ArgoCD Image Updater you're using :thinking:

Using the test-command inside the kubernetes cluster was broken. This should have been fixed by #293, but I think those changes haven't been released yet :thinking: cc @jannfis

Did you restart the pod after adding the registry configuration to your ArgoCD Image Updater installation?

**Comment by nishit93-hub** [comment]
@janpieper Thanks for the reply.

I synced with @jannfis on slack and he helped me with the issue.

I didn't restart the pod actually.
